### PR TITLE
Ari-patch-2

### PR DESCRIPTION
**Pull request by auriee**

### CODE REVIEW

File: draft-ar-pquip-pqc-engineers.md [hunk 1]
```diff
@@ -231,9 +231,9 @@ These algorithms are not a drop-in replacement for classical asymmetric cryptogr
 The fourth-round of the NIST process focuses only on KEMs. The goal of that round is to select an althernative algorithm that is based on different hard problem than Kyber.
 The candidates still advancing for standardization are:
 
-* [Classic McEliece](https://classic.mceliece.org/)
-* [BIKE](https://bikesuite.org/)
-* [HQC](http://pqc-hqc.org/)
+* [Classic McEliece](https://classic.mceliece.org/): Based on the hardness of syndrome decoding of Goppa codes
```
Comment (tireddy2) [inline draft-ar-pquip-pqc-engineers.md:234]: I suggest a line about Goppa code. For instance, we can say "Goppa codes are a class of error-correcting codes that can correct a certain number of errors in a transmitted message. The decoding problem involves recovering the original message from the received noisy codeword."

File: draft-ar-pquip-pqc-engineers.md [hunk 2]
```diff
@@ -231,9 +231,9 @@ These algorithms are not a drop-in replacement for classical asymmetric cryptogr
...
+* [BIKE](https://bikesuite.org/): Based on the the hardness of syndrome decoding of QC-MDPC codes.
```
Comment (tireddy2) [inline draft-ar-pquip-pqc-engineers.md:235]: Please add a line or two on the use QC-MDPC code.

File: draft-ar-pquip-pqc-engineers.md [hunk 3]
```diff
@@ -231,9 +231,9 @@ These algorithms are not a drop-in replacement for classical asymmetric cryptogr
...
+* [HQC](http://pqc-hqc.org/) : Based on the hardness of syndrome decoding of Hamming Quasi-cyclic codes.
```
Comment (tireddy2) [inline draft-ar-pquip-pqc-engineers.md:236]: Same comment as above.

**Review by tireddy2 — APPROVED** [review]

**Review by tireddy2 — APPROVED** [review]

**Review by tireddy2 — APPROVED** [review]

File: draft-ar-pquip-pqc-engineers.md [hunk 4]
```diff
@@ -231,9 +231,9 @@ These algorithms are not a drop-in replacement for classical asymmetric cryptogr
...
 The candidates still advancing for standardization are:
 
-* [Classic McEliece](https://classic.mceliece.org/): Based on the hardness of syndrome decoding of Goppa codes
-* [BIKE](https://bikesuite.org/): Based on the the hardness of syndrome decoding of QC-MDPC codes.
-* [HQC](http://pqc-hqc.org/) : Based on the hardness of syndrome decoding of Hamming Quasi-cyclic codes.
+* [Classic McEliece](https://classic.mceliece.org/): Based on the hardness of syndrome decoding of Goppa codes. Goppa codes are a class of error-correcting codes that can correct a certain number of errors in a transmitted message. The decoding problem involves recovering the original message from the received noisy codeword.
+* [BIKE](https://bikesuite.org/): Based on the the hardness of syndrome decoding of QC-MDPC codes. Quasi-Cyclic Moderate Density Parity Check (QC-MDPC) code are a class of error correcting codes that leverages bit flipping technique to efficiently correct the Hamming weight proportional to square root of the code length.
```
Comment (tireddy2) [inline draft-ar-pquip-pqc-engineers.md:235]: I would re-phrase the last line of BIKE as follows: Quasi-Cyclic Moderate Density Parity Check (QC-MDPC) code are a class of error correcting codes that leverages bit flipping technique to efficiently correct errors." You can add more details after this line or stop.

Comment (auriee) [reply draft-ar-pquip-pqc-engineers.md:235]: Sounds good.

File: draft-ar-pquip-pqc-engineers.md [hunk 5]
```diff
@@ -231,9 +231,9 @@ These algorithms are not a drop-in replacement for classical asymmetric cryptogr
...
+* [HQC](http://pqc-hqc.org/) : Based on the hardness of syndrome decoding of Quasi-cyclic BCH codes. BCH are a class of cyclic error-correcting codes that are constructed using polynomials in the Galois field and provide a great generalization of the Hamming code for multiple bit-error correction.
```
Comment (tireddy2) [inline draft-ar-pquip-pqc-engineers.md:236]: I propose the following minor update: 
BCH codes are a class of cyclic error-correcting codes constructed using polynomials in the Galois field. They offer a significant generalization of the Hamming code, enabling the correction of multiple bit errors.

**Review by tireddy2 — COMMENTED** [review]
My review

File: draft-ar-pquip-pqc-engineers.md (hunk 5 above)
Comment (auriee) [reply draft-ar-pquip-pqc-engineers.md]: Addressed.

**Pull request merged**